src/actions/breakpoints/remapLocations.js

```diff
@@ -2,8 +2,13 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
 
-export default function remapLocations(breakpoints, sourceId, sourceMaps) {
-  const sourceBreakpoints = breakpoints.map(async breakpoint => {
+ // @flow
+
+ import type { Breakpoint } from "../../types";
+ import type { BreakpointsMap } from "../../selectors";
```

Comment:
@darkwing i was getting issues with `valueSeq` thats why i did this , can you help me if there is any better way to solve it